helm_deploy/values-dev.yaml

```diff
@@ -1,18 +1,18 @@
----
-# Per environment values which override defaults in court-hearing-event-receiver/values.yaml
+minReplicaCount: 1
+maxReplicaCount: 2
 
-generic-service:
-  replicaCount: 2
+env:
+  JAVA_OPTS: "-Xmx512m"
 
-  ingress:
-    host: court-hearing-event-receiver-dev.hmpps.service.justice.gov.uk
+image:
+  repository: quay.io/hmpps/court-hearing-event-receiver
+  tag: latest
+  port: 8080
 
-  env:
-    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
-
-generic-prometheus-alerts:
-  alertSeverity: digital-prison-service-dev
```

Comment:
For this our team actual has it's own 'severity' which is `prepare-a-case` so if you set it to that we ought to receive these alerts whatever they are.